static-site/src/components/ListResources/Showcase.tsx

```diff
@@ -44,14 +45,17 @@ export const Showcase = ({cards, setSelectedFilterOptions}: ShowcaseProps) => {
 
   return (
     <div>
-      <Byline>
-        Showcase resources: click to filter the resources to a pathogen
-      </Byline>
       <ShowcaseContainer className={!isExpandable ? "" : isExpanded ? "expanded" : "collapsed"} $expandedHeight={cardsContainerHeight}>
         <CardsContainer ref={cardsContainerRef}>
-          {cards.map((el) => (
-            <ShowcaseTile card={el} key={el.name} setSelectedFilterOptions={setSelectedFilterOptions}/>
-            ))}
+          {cards.map((el: FilterCard | UrlCard) => {
+            if (Object.hasOwn(el, "filters")) {
+              return <FilterShowcaseTile card={el as FilterCard} key={el.name} setSelectedFilterOptions={setSelectedFilterOptions}/>
+            }
+            else if (Object.hasOwn(el, "url")) {
+              return <UrlShowcaseTile card={el as UrlCard} key={el.name} />
+            }
+            return
```

Comment:
might be worth doing this branch as something like:

```suggestion
            else { 
              throw new Error("this should never happen");
            }
```

also, i wonder if it's worth adding a `type` field to `FilterCard` and `UrlCard` and then doing this as a [discriminated union](https://gibbok.github.io/typescript-book/book/discriminated-unions/) kinda joint.

Reply:
Error handling added in 687bf223.

Reply:
> i wonder if it's worth adding a `type` field to `FilterCard` and `UrlCard` and then doing this as a [discriminated union](https://gibbok.github.io/typescript-book/book/discriminated-unions/) kinda joint.

This would need to be added in `static-site/content/resource-listing.yaml`:

```yaml
  - {name: "SARS-CoV-2",        img: "ncov.png",              filters: ["ncov"], type: "FilterCard"}
```

or injected based on presence of `filter`/`url`:

```js
if (Object.hasOwn(el, "filters")) {
  el.type = "FilterCard"
}
else if (Object.hasOwn(el, "url")) {
  el.type = "UrlCard"
}
else {
  throw new Error("this should never happen");
}
```

Both seem redundant, and I would rather check directly where the distinction is necessary: here, when deciding if a `FilterShowcaseTile` or `UrlShowcaseTile` should be displayed.

Reply:
1414985c92488423035a1470534c22a8ef479595 removed the need for this block of conditionals so nothing to be done here unless that is reverted.